.github/workflows/nv-transformers-v100.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       - name: Install pytorch
         run: |
           # use the same pytorch version as transformers CI
```

Comment:
@stas00 - could you point me to the place in transformers CI where this is set?  It looks like these are failing due to using torch 2.1.0 in place of the 2.0.1 we specify, but I wanted to make sure we were still on the same torch version.

Reply:
It looks like you have sorted it out already, Logan.

Also for future similar requests please tag @pacman100 who has taken over the maintenance of the Deepspeed integration and I'm not longer with HF. Thank you!

Reply:
I did - and will update who I tag in the future.  My apologies for the incorrect tag!

Reply:
no problem at all, Logan - you couldn't have known ;) 

But please feel free to tag me on other non-HF-specific issues - I still care very much for Deepspeed and hope to start using it again soon (current internode speed is too slow to permit ZeRO).